tests/python/contrib/test_cublas.py

```diff
@@ -73,11 +132,14 @@ def verify(target="cuda"):
     verify()
 
 def test_matmul_add():
-    verify_matmul_add('float', 'float')
+    verify_matmul_add('float', 'float', rtol=1e-3)
```

Comment:
Why do you need to change the `rtol` here?

Reply:
I tested it on Tesla T4 GPU with CUDA 10.1 and this one fails due to accuracy without rtol. I wonder if you can reproduce it on your end.

Reply:
Tested on GTX 2060 with CUDA 10.1, rtol=1e-5 works. I can remove rtol here.

Reply:
@masahi I'm not sure why it doesn't work with Tesla T4 using CUDA 10.1. I got the following error without rtol.
```
AssertionError:
Not equal to tolerance rtol=1e-05, atol=1e-07

(mismatch 74.61930614406779%)
 x: array([[513169.28, 508854.44, 480570.22, ..., 465465.12, 497088.88,
        532587.06],
       [546707.1 , 577276.  , 550151.9 , ..., 534576.9 , 574998.56,...
 y: array([[513163.28, 508860.03, 480572.5 , ..., 465454.3 , 497090.  ,
        532586.3 ],
       [546691.2 , 577286.06, 550138.5 , ..., 534561.75, 574991.1 ,...
```

Reply:
Its ok to leave rtol here as it is. Please address my last comment and i will merge.